api/checkin.ts

```diff
@@ -0,0 +1,68 @@
+import { config as dotenvConfig } from "dotenv-safe"
```

Comment:
is there any plain to use checkin outside `api.ts` ? 

Don't seem necessary to put on a different file.